src/ngx_http_lua_ssl_module.c

```diff
@@ -142,11 +142,11 @@ ngx_http_lua_ssl_merge_srv_conf(ngx_conf_t *cf, void *parent, void *child)
 
 
 int
-ngx_http_lua_ffi_get_ssl_ctx_count(ngx_cycle_t *cycle)
+ngx_http_lua_ffi_get_ssl_ctx_count()
```

Comment:
Should use `(void)` here instead. In the C language, they are very different things.

Reply:
I have modified and committed